packages/components/src/components/button/_button.scss

```diff
@@ -121,19 +122,31 @@
       background: transparent;
       border-color: transparent;
 
-      .#{$prefix}--btn__icon {
+      .#{$prefix}--btn__icon path {
         fill: $disabled;
       }
     }
 
     &.#{$prefix}--btn--sm {
-      padding: 0.375rem 1rem;
+      padding: $button-padding-ghost-sm;
     }
   }
 
+  .#{$prefix}--btn--icon-only {
+    @include tooltip--icon;
+  }
+
+  .#{$prefix}--btn--icon-only--top {
+    @include tooltip--icon--top;
+  }
+
+  .#{$prefix}--btn--icon-only--bottom {
```

Comment:
Is there ever going to be support for adding the tooltip to the left/right of the button.  Some use-cases have the button far aligned and the text may overflow past the viewport.

Reply:
I don't think there's an argument against it but it might be outside the scope of this PR since this is only bringing in the currently supported tooltip positions. I'm on board with rolling that into this PR if we're alright with the scope/feature creep

Reply:
Ideally I'd love for it it be in this PR but I understand scope creep and don't want to add to that so I'm okay if it's not part of this PR but please don't close the issue about the icon only button enhancement. We are tracking these issues within our own repo so we know when we can update to the Carbon code and remove our custom overwrites which we will not be able to do without the right/left implementation.